### PR TITLE
ci: add Node.js v16

### PR DESCRIPTION
**Pull request by Divlo**
**What is the purpose of this pull request? (put an "X" next to item)**

[ ] Documentation update
[ ] Bug fix
[ ] New feature
[x] Other, please explain: GitHub Actions change

**What changes did you make? (Give an overview)**

Add Node.js v16 to the matrix of `node-version`.

### CODE REVIEW

**Comment by voxpelli** [comment]
Should we temporarily remove the `bitmidi.com` repo? To enable us to merge this? As the failure is unrelated to this repo

**Comment by Divlo** [comment]
> Should we temporarily remove the `bitmidi.com` repo? To enable us to merge this? As the failure is unrelated to this repo

Hmm, yes so we can test on Node.js v16 sooner. :+1: 
Done!

We would need to don't forget to enable again [bitmidi.com](https://github.com/feross/bitmidi.com) when https://github.com/feross/bitmidi.com/pull/117 has been merged.

**Comment by Divlo** [comment]
Unrelated: Do you know why the badges are not showing for `test-external` and `test-internal` workflows ? @voxpelli 
That's probably a mistake I did on #1680. 😅

**Comment by voxpelli** [comment]
Easier to find the correct badge paths now after the merge:

Skärmavbild 2021-08-14 kl  21 16 49

> We would need to don't forget to enable again bitmidi.com when feross/bitmidi.com#117 has been merged.

Lets remember, we could add yet another PR here as well, but that feels a bit excessive 🙂

**Review by voxpelli — APPROVED** [review]

File: test/external/test.json [hunk 1]
```diff
@@ -1049,7 +1049,8 @@
   {
     "name": "bitmidi.com",
     "repo": "https://github.com/feross/bitmidi.com",
-    "install": true
+    "install": true,
+    "disable": true
```
Comment (voxpelli) [inline test/external/test.json:1053]:
```suggestion
    "disable": true // Disabled until fixed on Node 16: https://github.com/feross/bitmidi.com/pull/117
```

**Pull request merged**